frontend/workflows/experimentation/src/list-experiments.jsx

```diff
@@ -9,26 +42,27 @@ const ListExperiments = () => {
       setExperiments(response?.data?.experiments || []);
     });
 
-    return <Table headings={["Downstream Cluster", "Upstream Cluster"]} />;
+    return (
+      <Table headings={["Downstream Cluster", "Upstream Cluster", "Percentage", "HTTP Status"]} />
+    );
   }
 
   return (
-    <Table
-      data={experiments}
-      headings={["Downstream Cluster", "Upstream Cluster", "Percentage", "HTTP Status"]}
-    >
-      {experiments.map(e => (
-        <Row
-          key={e.id}
-          data={[
-            e.testSpecification.abort.clusterPair.downstreamCluster,
-            e.testSpecification.abort.clusterPair.upstreamCluster,
-            e.testSpecification.abort.percent,
-            e.testSpecification.abort.httpStatus,
-          ]}
-        />
-      ))}
-    </Table>
+    <Container>
+      <Table
+        data={experiments}
+        headings={["Downstream Cluster", "Upstream Cluster", "Percentage", "HTTP Status"]}
+      >
+        {experiments.map(e => {
```

Comment:
I'd like to add a 'stop experiment' button to each row in this table. Could you help me figure out how to do this? Should I add a Wizard to stop experiments and call that?

Reply:
You'd have to add another value to your mapped data that is just a `Button` which onClick calls out to a function to stop the experiment. I believe what you want is the data-layout with the wizard, you just won't use the wizard steps.

Reply:
I'll add this in a separate PR.